engine/src/main/java/org/terasology/rendering/dag/stateChanges/EnableStateParameter.java

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright 2017 MovingBlocks
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.terasology.rendering.dag.stateChanges;
+
+import static org.lwjgl.opengl.GL11.glEnable;
+
+/**
+ * TODO: Add javadocs
+ */
+abstract class EnableStateParameter extends SetStateParameter {
```

Comment:
Ah yes, of course. We didn't have this problem earlier because the task classes were segregating the glEnable/glDisable calls. Ok, please go ahead with the removal of the SetStateParameter class in favor of two Enable/DisableStateParameter classes.